hazelcast/src/main/java/com/hazelcast/collection/impl/collection/CollectionContainer.java

```diff
@@ -39,6 +39,10 @@
 @SuppressWarnings("checkstyle:methodcount")
 public abstract class CollectionContainer implements IdentifiedDataSerializable {
 
+    public static final int INVALID_ITEM_ID = -1;
+
+    public static final int ID_PROMOTION_OFFSET = 100000;
```

Comment:
This offset was introduced by Jaromir for the `IQueue` to create a buffer for in-flight operations, so we ensure that there are no duplicate itemIDs being created.

I put these commonly used constants to `CollectionContainer`, which is directly extended by `ISet` and `IList`. Its generic name also fits well for `IQueue`.